frontend/packages/dev-console/src/components/pipelines/modals/common/SecretAnnotation.scss

```
@@ -0,0 +1,4 @@
+.odc-secret-annotation__label {
```

```suggestion
.odc-secret-annotation {
    &__label {
```
Based on my other comment, I think this is more desirable. That way we get the structure we want for expanding and less noise when modifying / adding.